### ISSUE TITLE
2022 Period 1 Searchbox query problems

### ISSUE THREAD

**Issue by karlcz** [nih-cfde/cfde-deriva#353]
Multiple problems have been reported in the searchbox:
- searchbox inputs like `human` or `hum` leading to timeouts  (very common matches)
- searchbox inputs like `abcdefzzzzzz` leading to timeouts (something with zero matches)
- searchbox inputs of record-specific accession IDs leading to timeouts (because they are not searchable and hit same case as previous)

To improve this situation, we have to revisit the structure of the text search materials and indexing used by the searchbox interface. We also have found that we need to revise the postgres tuning parameters we established in a prior quarter, _so we also need to test for other portal performance regressions!_

Searchbox things to re-test for timeouts:
- [x] searchbox for common substrings like `human` and `hum`
- [x] searchbox for less frequent substrings
- [x] searchbox for unique substrings like file or biosample accession IDs
- [x] searchbox for non-matching substrings
- [x] basic facet selections using vocab
- [x] common fund program facet selections
- [x] combinations of facets
- [x] combinations of facets and searchbox
- [x] bdbag export option for different query forms above (assuming query itself doesn't already fail)

**Comment by karlcz** [comment]
My manual testing has found this to be a big improvement over the current prod system.  Some UI elements have noticable delays with spinners, but I have not reproduced the timeout errors that are easily triggered on the prod system.

It would be good to have others test different scenarios they are interested in, but I think I have tested the initial variants that I anticipated would tickle different behaviors from the DB.  This is merged for continued QA with intent to include these changes in the next release.
